test/fixtures/user/user.js

```diff
@@ -144,6 +144,9 @@ module.exports = () => {
         app_owner: true,
         archived: true,
       },
+      colors: {
+        color_id: 2,
```

Comment:
If its already a color object, then we can call it id itself instead of color_id?

Reply:
resolving this duplicate comment